extensions/huawei/GLSL_HUAWEI_subpass_shading.txt

```diff
@@ -0,0 +1,86 @@
+Name
+
+    HUAWEI_subpass_shading
+
+Name Strings
+
+    GL_HUAWEI_subpass_shading
+
+Contact
+
+    Hueilong Wang (wyvernathuawei), HUAWEI
+
+Contributors
+
+    Hueilong Wang, HUAWEI
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date: 2021-04-26
+    Revision: 1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 4.60
+    (#version 460) and higher.
+
+    This extension is written against revision 7 of the OpenGL Shading Language
```

Comment:
You don't give any interactions with the ES version of the spec. Is it your intention that the ES version of the shading language should be compatible with these shaders?

Reply:
Okay, I added them in https://github.com/KhronosGroup/GLSL/pull/163/commits/1114f6bed5d92f19481adecdad26a081fa1b4e09